tutorials/Land/Soil/interpolation_helper.jl

```diff
@@ -0,0 +1,22 @@
+function create_interpolation_grid(xbnd, xres, thegrid)
```

Comment:
Can we add docs? And should this live in `SingleStackUtils`?

Reply:
added docs. let's discuss the where it should live when we meet.